cluster-install/install.sh

```diff
@@ -2450,13 +2533,19 @@ step_generate_and_apply_resource_limit_patches() {
     # itself does "support" resource limits, in the sense that they don't get cleared out when we set them.
 
     # Trident Operator
-    if ! trident_will_be_installed_or_modified; then
+    if [ -z "$_USER_ACCEPTED_TRIDENT_OPERATOR_UPGRADE" ]; then
+        logdebug "no trident upgrade operator was needed, skipping trident operator resource limits"
+    elif [ "$_USER_ACCEPTED_TRIDENT_OPERATOR_UPGRADE" != "true" ]; then
+        logdebug "user declined trident operator upgrade, skipping trident operator resource limits"
```

Comment:
This is where we check if the user declined the upgrade, and block applying resource limits if so.